tools/plugins/com.liferay.ide.project.ui/META-INF/MANIFEST.MF

```diff
@@ -53,7 +53,9 @@ Require-Bundle: org.eclipse.ui,
  org.eclipse.sapphire.ui.swt.xml.editor,
  org.eclipse.sapphire.osgi,
  org.eclipse.sapphire.modeling.xml,
- org.eclipse.core.filesystem
+ org.eclipse.core.filesystem,
+ org.codehaus.jackson.mapper,
```

Comment:
Don't add dependencies like this, see Lovett's code where he uses jackson mapper for the Code Uprade stuff and already has the dependencies stuff worked out.